optimizely/project_config.py

```diff
@@ -191,51 +192,60 @@ def get_typecast_value(self, value, type):
         else:
             return value
 
+    def to_datafile(self):
```

Comment:
This should be called `get_datafile`

Reply:
@mjc1283 and I think it should be to_datafile on project config and get_datafile on optimizely config

Reply:
Why? `ProjectConfig` is an internal concept. And also, we are not doing any translation here.

Reply:
@mjc1283, do you have a strong opinion ?

Reply:
I like `to_datafile`. We aren't doing any translation, but if there is ever a `ProjectConfig` not backed by a datafile, we might do translation in that case. If @aliabbasrizvi strongly objects to `to_datafile`, I am OK with changing it to `get_datafile`.

Reply:
`to_datafile` is ok.